frame/staking/src/migrations.rs

```diff
@@ -70,10 +70,22 @@ pub mod v8 {
 
 pub mod v7 {
 	use super::*;
+	use frame_support::generate_storage_alias;
+
+	generate_storage_alias!(Staking, CounterForValidators => Value<u32>);
```

Comment:
this migration is not good, the module prefix is not fix with the value `Staking` on v6/v7.